app/assets/javascripts/custom.js

```diff
@@ -43,8 +50,16 @@ $(document).ready(function(){
     var food_id = $(this).data('message');
     var qty = parseFloat($(this).val());
     var price = parseFloat($('#food_sub_total_' + food_id).data('message'));
+    var prep_time = parseFloat($('#food_prep_' + food_id).data('message'));
```

Comment:
Identifier 'prep_time' is not in camel case.<br>Identifier 'food_id' is not in camel case.